test/colony-network-mining.js

```diff
@@ -1527,7 +1527,7 @@ contract("ColonyNetworkStaking", accounts => {
       const taskId = await setupRatedTask({
         colonyNetwork,
         colony: metaColony,
-        worker: MAIN_ACCOUNT,
+        worker: accounts[4],
```

Comment:
What's going on here?

Reply:
`MAIN_ACCOUNT` is manager, worker can't be a manager.

Reply:
There should be no restrictions on whether multiple roles can be the same person. We can't stop the same person having two different addresses after all.

Reply:
Ok, that shouldn't be hard to fix.